src/modules/DASDController.rb

```diff
@@ -742,9 +773,16 @@ def FormatDisks(disks_list, par)
       UI.CloseDialog
       iret = Convert.to_integer(SCR.Read(path(".process.status"), process_id))
       if iret != 0
-        # error report, %1 is exit code of the command (integer)
+        stderr = ""
+        loop do
+          line = SCR.Read(path(".process.read_line_stderr"))
+          break unless line
+          stderr << line
+        end
```

Comment:
I can live with some code duplication, but this looks as a missing opportunity to extract that loop into a `process_stderr` method.

Reply:
or `stderr_from_process` or whatever name fits you